pentaho/pom.xml

```diff
@@ -189,23 +191,26 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-module-junit4</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-core</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito</artifactId>
+      <version>5.10.0</version>
     </dependency>
     <!-- endregion -->
   </dependencies>
+
+  <build>
+    <plugins>
```

Comment:
Why was this added?

Reply:
I was getting errors "java.lang.ExceptionInInitializerError: Exception java.lang.reflect.InaccessibleObjectException: Unable to make protected final java.lang.Class" I tried using <maven-surefire-plugin.argLine> --add-opens=java.base/java.lang.reflect=ALL-UNNAMED </maven-surefire-plugin.argLine> in the properties section along with few more packages, still I couldnt build, but when I check this configuration block is from the pentaho-ce-parent pom and use this in the current pom file, then I am able to build.

Reply:
This pom should have inherited this from the pentaho-ce-parent pom file in the first place; that's why I'm confused why you added it here.  It doesn't make sense why this would solve the error you got since you're still not changing the --add-opens arguments anywhere in here.